src/components/DataArchives/DataArchives.js

```diff
@@ -88,25 +88,44 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const sensors = "https://api.sensors.africa/v1/sensors/{sensor_id}/";
-const query = "https://api.sensors.africa/v1/filter?city=&country=&type=";
-const data = "https://api.sensors.africa/v1/data/";
-const now = "https://api.sensors.africa/v1/now/";
-const type = "{sensor type}";
-const city = "{city}";
-const countryCode = "{country name}";
+const API_BASE_URL = process.env.NEXT_PUBLIC_APP_API_BASE_URL;
+const API_DOCS_URL = process.env.NEXT_PUBLIC_APP_API_DOCS_URL;
+const API_STATIC_URL = process.env.NEXT_PUBLIC_APP_API_STATIC_URL;
 
-const allCities = "http://api.sensors.africa/v2/cities";
-const specificCities = "https://api.sensors.africa/v2/data/air?city={slug}&";
-const nodes = "https://api.sensors.africa/v2/nodes";
-const data5 = "http://api.sensors.africa/static/v2/data.json";
-const data1 = "http://api.sensors.africa/static/v2/data.1h.json";
-const data24 = "http://api.sensors.africa/static/v2/data.24h.json";
-const dataDust = "http://api.sensors.africa/static/v2/data.dust.min.json";
-const otherSensors = "http://api.sensors.africa/static/v2/data.temp.min.json";
-
-const toFrom = "{YYYY - mm - dd}";
-const valueType = "{P1, P2, temperature, humidity}";
+const API_CONFIG = {
+  endpoints: {
+    sensors: `${API_BASE_URL}/sensors/{sensor_id}/`,
+    query: `${API_BASE_URL}/filter?city=&country=&type=`,
+    data: `${API_BASE_URL}/data/`,
+    now: `${API_BASE_URL}/now/`,
+    allCities: `${API_BASE_URL}/cities`,
+    specificCities: `${API_BASE_URL}/data/air?city={slug}&`,
+    nodes: `${API_BASE_URL}/nodes`,
+  },
+  docs: {
+    sensors: `${API_DOCS_URL}sensors-list`,
+    query: `${API_DOCS_URL}/filter?city=&country=&type=`,
+    data: `${API_DOCS_URL}data-stats-list`,
+    now: `${API_DOCS_URL}now-list`,
+    allCities: `${API_DOCS_URL}cities-list`,
+    nodes: `${API_DOCS_URL}nodes-list_nodes`,
+  },
+  staticData: {
+    data1: `${API_STATIC_URL}/data.1h.json`,
+    data5: `${API_STATIC_URL}/data.json`,
+    data1h: `${API_STATIC_URL}/data.1h.json`,
+    data24h: `${API_STATIC_URL}/data.24h.json`,
+    dataDust: `${API_STATIC_URL}/data.dust.min.json`,
+    otherSensors: `${API_STATIC_URL}/data.temp.min.json`,
+  },
+  placeholders: {
+    type: "{sensor type}",
```

Comment:
Mostly looks good, just one additional suggestion,  on how the API_CONFIG should look like,

I could be wrong but wouldn't something like this be more maintainable ? 

```
const API_CONFIG = {
     sensors: {
      endpoint: `${API_BASE_URL}/sensors/{sensor_id}/`
      docs: `${API_DOCS_URL}/sensors-list`,
    },
   ...
}

```

Also double check how the URLs are being concatenated there are places where there is a missing `/`, could be problematic e.g. is this intended:

```
docs: {
    sensors: `${API_DOCS_URL}sensors-list`,
    query: `${API_DOCS_URL}/filter?city=&country=&type=`,
    data: `${API_DOCS_URL}data-stats-list`,
```

Reply:
Hi @m453h on this

> Also double check how the URLs are being concatenated there are places where there is a missing /, could be problematic e.g. is this intended:

The docs base url is `'https://staging.api.sensors.africa/docs/#v2-'` so the concatenation should be as `sensors`